glue/configure_includes/configure_win_x86.mk

```diff
@@ -111,3 +111,17 @@ CONFIGURE_ARGS += 'OMR_HOST_OS=win'
 CONFIGURE_ARGS += 'OMR_HOST_ARCH=x86'
 CONFIGURE_ARGS += 'OMR_TARGET_DATASIZE=$(TEMP_TARGET_DATASIZE)'
 CONFIGURE_ARGS += 'OMR_TOOLCHAIN=msvc'
+
+COPYRIGHT_WIN_COMPANYNAME:=International Business Machines Corporation\0
```

Comment:
What happens if these values are not set?

Also you these values should not be set to J9 values.  If there needs to be a default value then it should be an OMR default value.